web-client/src/components/SideDrawer/SideDrawer.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  CloseOutlined,
+  LogoutOutlined,
+  MailOutlined,
+  StarOutlined,
+  TeamOutlined,
+} from '@ant-design/icons';
+import { Layout, Typography } from 'antd';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { COLORS } from 'src/theme/colors';
+import { MenuItem } from 'src/types/menu-item';
+import styled from 'styled-components';
+
+import SideNavMenu from '../SideNavMenu/SideNavMenu';
+
+const { Text } = Typography;
+
+const SideDrawer: React.FC<SideDrawerProps> = ({
+  collapsed,
+  closeSider,
+  menuItems,
+  profileData,
+  siteLocations,
+}) => (
+  <Layout.Sider collapsed={collapsed} theme="light">
+    <CloseButton onClick={closeSider}>
+      <CloseOutlined />
+    </CloseButton>
+    <ProfileWrapper>
+      <ProfileImg src={profileData.photoUrl} />
+      <ProfileContent>
+        <ProfileUserName>{profileData.fullName}</ProfileUserName>
+        <ProfileDetails>
+          <ProfileDetail>
+            <TeamOutlined />
+            {profileData.followers}
+          </ProfileDetail>
+          <ProfileDetail>
+            <StarOutlined />
+            {profileData.stars}
+          </ProfileDetail>
+        </ProfileDetails>
+      </ProfileContent>
+    </ProfileWrapper>
```

Comment:
This can be extracted to its own component

Reply:
Yeah. Can you take care of that?

Reply:
Done by 72464e5